style.css

```diff
@@ -0,0 +1,51 @@
+body {
+    background-color: black;
+  }
+  
+  nav {
+    display: flex;
+  }
+  
+  main {
+    display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` is commonly used as a property in the style sheet, instead we can create a class with the property and assign it when needed to a class in our HTML code, here is an example:

```
.flex-box {
  display: flex;
}
```